lib/MDButton.css

```diff
@@ -10,10 +10,11 @@
 .md-btn-default, .md-btn-primary, .md-btn-info, .md-btn-danger {
   display: inline-block;
   margin: 5px 0px;
-  padding: 6px 12px;
-  border-width: 1px;
+  padding: 5px 30px;
+  border-width: 0;
+  box-shadow: 0px 0px 1px 0px rgba(0,0,0,0.50);
   border-style: solid;
-  border-radius: 4px;
+  border-radius: 0;
```

Comment:
right. I'd rather keep it. so that I can find out the exact border-radius instead of inferring from border: none.